src/client/components/collection/Badges.js

```diff
@@ -26,11 +25,9 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function Badges({ views, likes, rating }) {
+function Badges({ views, likes }) {
```

Comment:
Why are ratings not included yet?

Reply:
André said that since most of the ratings are not set, it was better not displaying them rather than displaying 0-rated collections.